src/utils/network/session-core.js

```diff
@@ -809,8 +547,11 @@ export const sendMessageTTL = async (session, messageObject) => {
 
 export const createOutbox = async (session) => {
   const outboxContainerUrl = getContainerUrl(session, 'Outbox', 'self-fetch');
-  await createContainerAt(outboxContainerUrl, { fetch: session.fetch });
+  try {
```

Comment:
The try/catch approach in PR #201 should be enough to fix this function.

Reply:
Removed this